uftrace.c

```diff
@@ -235,13 +236,13 @@ __used static const char uftrace_footer[] =
 "\n";
 
 static const char uftrace_shopts[] =
-	"+aA:b:C:d:D:eE:f:F:hH:kK:lL:N:P:r:R:s:S:t:T:U:vVW:Z:";
+	"+aA:b:C:d:D:eE:f:F:hH:kK:lN:P:r:R:s:S:t:T:U:vVW:Z:";
 
 #define REQ_ARG(name, shopt) { #name, required_argument, 0, shopt }
 #define NO_ARG(name, shopt)  { #name, no_argument, 0, shopt }
 
 static const struct option uftrace_options[] = {
-	REQ_ARG(library-path, 'L'),
+	REQ_ARG(libmcount-path, OPT_libmcount_path),
 	REQ_ARG(filter, 'F'),
 	REQ_ARG(notrace, 'L'),
```

Comment:
Unrelated, but I think the short option for notrace should 'N'.  Not sure how it worked so far..

Reply:
:) It's another magic.. I fixed it in split patch for unexpected results.
Please let me know if you want squash it to first patch.
Thank you.